### PR TITLE
Enhance/prevent load of unnecessary scripts styles

### LINKED ISSUE #1334: WP Rocket is giving error when Delay JS Execution on on Single Product Page
On `dokan-lite/includes/Assets.php` file All dokan js are loading on Single product page. There should e filter so we can remove dokan js form single product page.  http://prntscr.com/1t2ajz0

When I removed the `is_product()` check there is no error from WP Rocket

### PR DESCRIPTION
**Pull request by NAbdulla1**
### All Submissions:

* [x] My code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)
* [x] My code satisfies feature requirements
* [x] My code is tested
* [x] My code passes the PHPCS tests
* [x] My code has proper inline documentation
* [ ] I've included developer documentation (optional)
* [x] I've added proper labels to this pull request

### Changes proposed in this Pull Request:
Stopped loading of certain scripts sitewide. Now those scripts loads on those pages where they are necessary.

this pr is part of [this](https://github.com/weDevsOfficial/dokan-pro/pull/1850) dokan pro pr
Closes #1334 
### How to test the changes in this Pull Request:

1. take a look at [this](https://docs.google.com/spreadsheets/d/1kh2vAXHQyt41gsJkWKt4xgTqK1qdIf0SimhEZBGISOc/edit#gid=1792838472) excel sheet
2. check if the handles specified load sitewide or not
3. check if the handles are loading on required pages by looking at the documentation of Dokan

### Changelog entry

> **enhance:** stop loading unnecessary scripts and styles sitewide

### PR Self Review Checklist:

* Code is not following code style guidelines
* Bad naming: make sure you would understand your code if you read it a few months from now.
* KISS: Keep it simple, Sweetie (not stupid!).
* DRY: Don't Repeat Yourself.
* Code that is not readable: too many nested 'if's are a bad sign.
* Performance issues
* Complicated constructions that need refactoring or comments: code should almost always be self-explanatory.
* Grammar errors.

### CODE REVIEW

File: includes/Assets.php [hunk 1]
```diff
@@ -564,8 +567,6 @@ public function enqueue_front_scripts() {
             }
         }
 
-        wp_enqueue_script( 'dokan-login-form-popup' );
```
Comment (tanmayjay) [inline includes/Assets.php:567]: Isn't this pop up needed?

**Review by tanmayjay — CHANGES_REQUESTED** [review]

File: includes/Assets.php (hunk 1 above)
Comment (NAbdulla1) [reply includes/Assets.php:567]: this popup is needed when the user is not logged in and is trying to follow/unfollow store, get store support, report abuse. I have deleted it from here and enqueued it when necessary

**Review by tanmayjay — APPROVED** [review]

**Comment by Rat01047** [comment]
Issue-1:
Console Error on My-Account Page

Scenario:
Navigate to My-Account Page without logging in > Console Error 

Screenshot given for reference:
image

**Comment by shashwatahalder01** [comment]
Scenario:
Design break, product menu missing, review menu not working.

Screenshot:
sl1

previous view:
Sl2

**Comment by Rat01047** [comment]
Issue-5:
Rating of vendor is broken

Scenario:
Product Details > Click on Vendor info > Ratings are broken in design

Screenshot given for reference:
image

**Pull request merged**